draft-ietf-quic-transport.md

```diff
@@ -2357,11 +2361,11 @@ shown in the following figure and described below.
                             +--------+
                                  |
                         send/recv STREAM/RST
-                             recv MSD/SB
+                             recv MSD/SB             
                                  |
                                  v
-                 recv FIN/  +--------+    send FIN/
-                 recv RST   |        |    send RST
+              recv FIN/     +--------+     send FIN/
+         recv RST/send UNI  |        |  send RST/recv UNI
```

Comment:
I clarified the text to indicate only the peer the stream is reserved for can send the first STREAM frame.  In regards to RST, I added text on that as well, but I suspect it needs a state transition change as well, which I haven't fully thought through.  Feel free to offer a suggestion.